skt/executable.py

```diff
@@ -169,8 +169,13 @@ def cmd_build(cfg):
     tstamp = datetime.datetime.strftime(datetime.datetime.now(),
                                         "%Y%m%d%H%M%S")
 
-    builder = skt.kbuilder(cfg.get('workdir'), cfg.get('baseconfig'),
-                           cfg.get('cfgtype'), cfg.get('makeopts'))
+    builder = skt.kbuilder(
+        cfg.get('workdir'),
+        cfg.get('baseconfig'),
+        cfg.get('cfgtype'),
+        cfg.get('makeopts'),
+        cfg.get('enable_debuginfo')
+    )
```

Comment:
This is getting a little out of hand. We'll need to convert this to named arguments next.

Reply:
Agreed. I can work on that.